### PR TITLE
Adding Latin subset of Lato font

### PR DESCRIPTION
**Pull request by baohouse**
Homepage by default primarily uses Latin scripts for Lato, so we'll optimize loading Latin-based Lato; any non-Latin characters will appropriately load the full set.

I had to use FontForge and manually inspect the Unicode ranges that the Latin-optimized and full set versions of the font contained, so that's where those numbers in the ranges come from.

Initial load of fonts on the homepage went from 1020 kB down to 289 kB (75 kB actually belongs to FontAwesome).

### CODE REVIEW

File: app/assets/stylesheets/base/_fonts.scss [hunk 1]
```diff
@@ -1,221 +1,477 @@
 // This file contains the base fonts for the application.
 
-$font-cdn-base: 'https://cdn.jsdelivr.net/font-lato/2.0/Lato';
+$full-set-cdn: 'https://cdn.jsdelivr.net/font-lato/2.0/Lato';
+$non-latin-range:
```
Comment (julianguyen) [inline app/assets/stylesheets/base/_fonts.scss]: Oh snap, you manually figured this out? Wow 🥇

Comment (baohouse) [reply app/assets/stylesheets/base/_fonts.scss]: In my own weird way, going through each character is like Tibetan monks making mandala sand art.

Comment (julianguyen) [reply app/assets/stylesheets/base/_fonts.scss]: Do you anticipate ever having to update this list?

Comment (baohouse) [reply app/assets/stylesheets/base/_fonts.scss]: By that time we will want to have automated deployment scripts to split the font by language. This was just a low-hanging fruit in that Lato officially released Latin vs full set versions of its font.

**Pull request merged**